### DISCUSSION TITLE
getting timeout 504 in 1 minute

### DISCUSSION THREAD

**Discussion by ravi160822** [tiangolo/uvicorn-gunicorn-fastapi-docker#324] [Questions] [answered]
only happens on dev, when I process a csv then does not happen in dev
what could be the reason for this?

**Comment by tiangolo** [comment, marked as answer]
It would probably be better to provide a simple example app that can be copy-pasted to replicate the problem. :thinking: 

But anyway, now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image
